canonical/loki-k8s-operator
# Build manual trigger for itests

## Issue

There are situations in which we believe `main` is broken, especially its integration tests breaking on external dependencies, but we have no means to triggering them to verify.

## Solution

Provide manual triggers for running itests, and reuse the itest job across various builds.

## Context

N/A

## Testing Instructions

Go to [Actions](https://github.com/canonical/loki-k8s-operator/actions) and see if you can trigger a run of itests for `main` :-)

## Release Notes

N/A
